_pages/how-we-work/sensitive-information.md

```diff
@@ -6,22 +6,24 @@ Here's what you need to know about sensitive information at 18F.
 
 ## What is considered sensitive
 
-To learn more about what information is considered sensitive, see [our Open Source Policy practices guide](https://github.com/18F/open-source-policy/blob/master/practice.md#protecting-sensitive-information).
+To learn what information we consider sensitive, see [our Open Source Policy practices guide](https://github.com/18F/open-source-policy/blob/master/practice.md#protecting-sensitive-information).
 
 ## Tools
 
-There are several options for sharing sensitive information at 18F. As you'll remember from your Security Awareness and Privacy training in [GSA Online University (OLU)](https://gsaolu.gsa.gov), please make sure to share sensitive information with only the people that absolutely need it, and who are authorized to see it.
+There are several options for sharing sensitive information at 18F. As taught in your Security Awareness and Privacy training in [GSA Online University (OLU)](https://gsaolu.gsa.gov), only share sensitive information with the people who absolutely need it and are authorized to see it.
 
 ### Fugacious
 
-<!-- Note this information needs to remain *somewhere* for cloud.gov FedRAMP compliance. -->
+<!-- Note this information needs to remain *somewhere* for cloud.gov FedRAMP compliance, since the cloud.gov team uses Fugacious. It's linked from https://cloud.gov/docs/ops/secrets/#sharing-secret-keys - if you make (or want to make) major changes here, please ping #cloud-gov-highbar. -->
 
-[Fugacious](https://fugacious.18f.gov/) is a tool for short-term secure messaging. Fugacious is most useful for sending small bits of text once, such as passwords or API keys. We recommend setting the lowest acceptable "self-destruct" settings for number of views and hours available that you need. Do not set the hour limit for more than 120 hours. Fugacious links can be shared over Slack, but please don't put sensitive information in Slack directly.
+You can use [Fugacious](https://fugacious.18f.gov/) for short-term secure messaging. Fugacious is most useful for sending small bits of text once, such as passwords or API keys. Set the lowest acceptable "self-destruct" settings for number of views and hours available that you need. Do not set the hour limit for more than 120 hours (it's currently configured with a maximum of 48 hours). You can share Fugacious links over TTS Slack, but don't put sensitive information in Slack directly.
```

Comment:
> Do not set the hour limit for more than 120 hours (it's currently configured with a maximum of 48 hours).

This is misleading; why not just dictate "do not set the hour limit for more than 48 hours"?